src/components/HtxTextBox/HtxTextBox.js

```diff
@@ -28,27 +28,47 @@ export class HtxTextBox extends React.Component {
   }
 
   componentDidMount() {
-    if (isFF(FF_DEV_1566)) {
-      window.addEventListener('click', this.handleGlobalClick, { capture: true });
-    }
+    this.focus();
+
+    window.addEventListener('click', this.handleGlobalClick, { capture: true });
   }
 
   componentWillUnmount() {
-    if (isFF(FF_DEV_1566)) {
-      window.removeEventListener('click', this.handleGlobalClick, { capture: true });
-    }
+    window.removeEventListener('click', this.handleGlobalClick, { capture: true });
   }
 
-  handleGlobalClick = (e) => {
-    const el = e?.target;
-    const isShortcut = el?.dataset?.shortcut;
-    const shouldSkip = !this.state.editing || this.props.ignoreShortcuts && isShortcut || el === this.inputRef.current;
+  clickedElement = (clickedElement, targetElement) => {
+    return clickedElement && (clickedElement === targetElement || clickedElement.contains(targetElement));
+  };
+
+  handleGlobalClick = e => {
+    const elementTarget = e?.target;
+    const playPauseButton = document.getElementById('play-pause');
+    const waveFormElement = document.getElementById('wave');
+    const clickedPlayPauseButton = this.clickedElement(playPauseButton, elementTarget);
+    const clickedWaveFormElement = this.clickedElement(waveFormElement, elementTarget);
 
-    if (!shouldSkip) {
-      this.setEditing(false);
+    if (clickedPlayPauseButton || clickedWaveFormElement) {
+      this.focusTextArea();
+      return;
+    }
+
+    if (isFF(FF_DEV_1566)) {
```

Comment:
Do we still need this feature flag?

Reply:
Also, I don't quite understand this section below. What's going on?

Reply:
This handles the global click i.e. when an area outside the text area is clicked the text area is blurred/ not focused anymore

Reply:
add comment, and remove feature flag